types/parse/index.d.ts

```diff
@@ -657,15 +668,23 @@ namespace Parse {
             X extends Extract<keyof U['attributes'], string>>(key: K, queryKey: X, query: Query<U>): this;
         doesNotMatchQuery<U extends Object, K extends keyof T['attributes']>(key: K, query: Query<U>): this;
         distinct<K extends keyof T['attributes'], V = T['attributes'][K]>(key: K): Promise<V>;
-        each(callback: (obj: T) => PromiseLike<void> | void, options?: Query.EachOptions): Promise<void>;
+        eachBatch(callback: (objs: T[]) => Promise<any>, options?: Query.BatchOptions): Promise<void>;
```

Comment:
Is there any reason for not using the previous return type?
```suggestion
        eachBatch(callback: (objs: T[]) => PromiseLike<void> | void, options?: Query.BatchOptions): Promise<void>;
```

It signals that you can either return a `Promise` or not, and that the value of the promise won't be used.